### PR TITLE
qa: set mds config with `config set` for a particular test

### PR DESCRIPTION
**Pull request by vshankar**
https://tracker.ceph.com/issues/57087

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/tasks/cephfs/test_data_scan.py [hunk 1]
```diff
@@ -491,10 +491,11 @@ def test_fragmented_injection(self):
 
         file_count = 100
         file_names = ["%s" % n for n in range(0, file_count)]
+        split_size = 100 * file_count
 
         # Make sure and disable dirfrag auto merging and splitting
-        self.fs.set_ceph_conf('mds', 'mds bal merge size', 0)
-        self.fs.set_ceph_conf('mds', 'mds bal split size', 100 * file_count)
+        self.fs.mon_manager.run_cluster_cmd(args='config set mds mds_bal_merge_size 0')
+        self.fs.mon_manager.run_cluster_cmd(args=f'config set mds mds_bal_split_size {split_size}')
```
Comment (dparmar18) [inline qa/tasks/cephfs/test_data_scan.py:498]:
```suggestion
        self.config_set('mds', 'mds_bal_merge_size', 0)
        self.config_set('mds', 'mds_bal_split_size', split_size)
```

**Comment by vshankar** [comment]
Done. Will squash the commit once the changes are approved.

**Review by lxbsz — APPROVED** [review]
LGTM.

**Comment by vshankar** [comment]
@mchangir for integration tests.

**Review by joscollin — APPROVED** [review]

**Review by dparmar18 — APPROVED** [review]

File: qa/tasks/cephfs/test_data_scan.py [hunk 2]
```diff
@@ -494,8 +494,8 @@ def test_fragmented_injection(self):
         split_size = 100 * file_count
 
         # Make sure and disable dirfrag auto merging and splitting
-        self.fs.mon_manager.run_cluster_cmd(args='config set mds mds_bal_merge_size 0')
-        self.fs.mon_manager.run_cluster_cmd(args=f'config set mds mds_bal_split_size {split_size}')
+        self.config_set('mds', 'mds_bal_merge_size', 0)
```
Comment (kotreshhr) [inline qa/tasks/cephfs/test_data_scan.py:497]: Is this the general practice to make the suggestion into different commit ? I don't have strong opinion here but  I generally  find it cluttered to have the suggestion a separate commit.

Comment (vshankar) [reply qa/tasks/cephfs/test_data_scan.py:497]: I used the UI to merge the suggestion and that creates an additional commit. I'll squash it before merging.

Comment (vshankar) [reply qa/tasks/cephfs/test_data_scan.py:497]: https://github.com/ceph/ceph/pull/54590#issuecomment-1820543576

**Review by batrick — APPROVED** [review]

**Comment by vshankar** [comment]
@rishabh-d-dave status of this tests?

**Review by rishabh-d-dave — APPROVED** [review]

**Review by rishabh-d-dave — COMMENTED** [review]
Testing was successful - https://tracker.ceph.com/projects/cephfs/wiki/Main#16-Jan-2023
